js/main.js

```diff
@@ -0,0 +1,27 @@
+var $imgPreview = document.querySelector('img');
+var $imgUrlField = document.querySelector('#img-src');
+var $form = document.querySelector('form');
+
+var livePreview = event => {
+  var $imgUrlInput = event.target.value;
+  $imgPreview.setAttribute('src', $imgUrlInput);
+};
+
+var logNewEntry = event => {
+  event.preventDefault();
+
+  var $newEntry = {
+    title: $form.elements.title.value,
+    photoUrl: $form.elements['img-src'].value,
+    notes: $form.elements['img-notes'].value,
+    entryId: data.nextEntryId
+  };
+
+  data.nextEntryId++;
+  data.entries.push($newEntry);
```

Comment:
You should use `unshift` here, so that the newest entry will always be at the beginning of the array. This will come in handy in the next feature when you render the entries